rdrf/rdrf/admin.py

```diff
@@ -452,6 +453,23 @@ class DashboardWidgetAdmin(admin.ModelAdmin):
     inlines = [DashboardLinksInline, DashboardDemographicsInline, DashboardCdeDataInline]
 
 
+class LongitudinalFollowupAdmin(admin.ModelAdmin):
```

Comment:
Can we use the feature flags to hide the admin options for Longitudinal Followup and Longitudinal Followup Entries?

Reply:
Done